subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitor.java

```diff
@@ -121,7 +132,7 @@ String getJstackCommand() {
         String jstack() {
             ExecResult result = run(getJstackCommand(), pid);
 
-            StringBuilder sb = new StringBuilder(String.format("Run %s %s return %s", getJstackCommand(), pid, result.toString()));
+            StringBuilder sb = new StringBuilder(String.format("Run %s %s return %s", getJstackCommand(), pid, result));
```

Comment:
We talked about using the new `ProcessHandle` api instead of running something on the command line. Any thoughts on that?

Reply:
Unfortunately, this doesn't work on windows because of https://bugs.openjdk.org/browse/JDK-8176725. I've tried that on Windows but all `arguments` returned by `ProcessHandle` API is empty.

Another option is `jps -lvm`, which can make the parser a bit simpler.